extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/JibConfig.java

```diff
@@ -161,16 +161,16 @@ public class JibConfig {
      * {@literal linux/amd64,linux/arm64/v8}
      * </pre>
      *
-     * If not specified, OS default is linux and architecture default is amd64
+     * If not specified, OS default is linux and architecture default is {@code amd64}.
      *
      * If more than one platform is configured, it is important to note that the base image has to be a Docker manifest or an
-     * OCI image index containing a version of each chosen platform
+     * OCI image index containing a version of each chosen platform.
      *
-     * It doesn't work with native images, as cross-compilation is not supported
+     * The feature does not work with native images, as cross-compilation is not supported.
      *
-     * Target Platform is a incubating feature of Jib. See <a href=
+     * This configuration is based on an incubating feature of Jib. See <a href=
```

Comment:
Although it is nice, this change is not related to the PR proposal, it should be in a separate commit